src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -2940,6 +2985,24 @@ private int MinBatchSize
                 }
             }
 
+            if (ScratchSpaceLimitInBytes != null && txAllocations > ScratchSpaceLimitInBytes)
+            {
+                stats.RecordMapCompletedReason($"Reached scratch space limit ({ScratchSpaceLimitInBytes / 1024:#,#0} kb). Allocated {txAllocations / 1024:#,#0} kb of scratch space in current transaction");
```

Comment:
Can you use size here?

Reply:
Sure. I did it this way for consistency with existing messages. Will fix it since I saw you changed it it another PR